ruma-events/tests/ui/08-enum-invalid-path.stderr

```diff
@@ -10,9 +10,3 @@ error: proc macro panicked
    | |_^
    |
    = help: message: well-known matrix events have to start with `m.` found `not.a.path`
-
```

Comment:
Did this test fail for you? It passes on stable (1.44) and beta (1.45). Either way, please undo this change.

Reply:
I think I had the same problem with a recent PR, but I might not have been updated to 1.44 on my laptop.

Reply:
My bad, I updated the toolchain and it passed.

Reply:
Good to know it's the toolchain version, in case it comes up again.

Reply:
@jplatte 
A suggestion for this: it might be worthwhile to standardize using Docker to `cargo build`, `cargo test`.
It gets rid almost all "This works on my PC" kind of problem

Reply:
@Kelerchian Thanks for the suggestion, but I think setting up Docker is more complicated than keeping `rustc` up to date for most people. This kind of issue rarely comes up, and when it does it's usually very easy to fix (like it was here).